test/json.go

```diff
@@ -0,0 +1,24 @@
+package test
+
+import (
+	"encoding/json"
+	"os"
+	"testing"
+)
+
+// ReadJSON is a testing utility that reads data from JSON file into an
```

Comment:
I think it might be nice to describe the purpose as well, so perhaps something like:

    ReadJSON decodes JSON from a file into a value. If it fails to open the given
    filename or decode the contents of the file, ReadJSON will call t.Fatal with the
    error.

    This is intended to be used in test functions where an expected result for a test
    is stored as a golden file alongside the tests.

    The type of the first parameter as testing.TB allows it to be used from both
    tests and benchmarks, passing either a *testing.T or *testing.B.

    <add code sample here>

Reply:
thanks.

Reply:
i am not sure if it is intended for golden files though. could just as well be used for input (I do that...).
i think it can be used in tests for inputs and outputs equally well...